nmdc_schema/nmdc_schema_generated.yaml

```diff
@@ -395,8 +395,10 @@ enums:
     permissible_values:
       Metagenome Raw Reads:
         text: Metagenome Raw Reads
-        todos:
-        - add annotations
+        annotations:
+          file_name_pattern:
+            tag: file_name_pattern
+            value: '/.+?(?=fastq)/fastq.gz'
```

Comment:
There's a compact way of specifying annotations in the schema, like below:

```yaml
annotations:
  file_name_pattern: '/.+?(?=fastq)/fastq.gz'
```

Which is how I think you should represent it, simply so you're consistent with the way in which the annotations are specified on the rest of the permissible values in the enum, and not because it's incorrect. It's verbose, and as correct as the compact solution, but the point I'm trying to highlight is consistency.

Reply:
Excellent point, @sujaypatil96 

I suggest we prioritize adding the content and therefore move forward with this merge. We can programmatically condense the modeling later. I'll make an issue

Reply:
Sounds good @turbomam. But I think @Michal-Babins still needs to update the PR so the changes are in `src/schema/nmdc.yaml` and not the current file where the changes have been made.